src/bin/typings-install.ts

```diff
@@ -120,7 +126,7 @@ function installer (args: Args & minimist.ParsedArgs) {
     function handle (options: InstallDependencyOptions) {
       return loader(installDependency(location, options), args)
         .then(output => {
-          printResult(output, { name: options.name })
+          printResult(output, { name: options.name, save: options.save, saveDev: options.saveDev, ambient: options.ambient })
```

Comment:
Can do - it's only the ambient, save and saveDev flags we're interested in (I think) and so I'll pass in full and check specifically in the function.